db/db.test.js

```diff
@@ -0,0 +1,21 @@
+const mongoose = require('mongoose');
+
+describe('DB Test', () => {
+  beforeAll((done) => {
+    mongoose.connect('mongodb://localhost/data');
+
+    let db = mongoose.connection;
+
+    db.on('error', (err) => {
```

Comment:
Wondering if there is a way to have this inside a test()?  It seems a little strange that the actual test is in beforeAll() and getting to test() indicates success...  I don't know much about mongoose testing though

Reply:
I read up to test for Mongodb/Mongoose, we have to use beforeAll to create the connection and once connection is established, it invokes the done function( in my code: line 18-20).